backend/internal/environment/environment.go

```diff
@@ -32,6 +32,18 @@ const (
 	// Environment variable used to configure the allowed origins for
 	// accessing the API.
 	EnvVarAPIAllowedOrigins = "ZEEVISION_API_ALLOWED_ORIGINS"
+	// Environment variable used to configure the database username
+	EnvVarDatabaseUser = "ZEEVISION_DATABASE_USER" //nolint:all
+	// Environment variable used to configure the database user password
+	EnvVarDatabasePassword = "ZEEVISION_DATABASE_PASSWORD" //nolint:all
+	// Environment variable used to configure the database name
+	EnvVarDatabaseName = "ZEEVISION_DATABASE_NAME"
+	// Environment variable used to configure if the database should be
+	// hosted.
+	EnvVarHostDatabase = "ZEEVISION_HOST_DATABASE"
```

Comment:
Is this comment description accurate? I think this was the host name or something, reading the README.